solr/core/src/java/org/apache/solr/schema/TextField.java

```diff
@@ -43,6 +43,7 @@
 public class TextField extends FieldType {
   protected boolean autoGeneratePhraseQueries;
   protected boolean enableGraphQueries;
+  protected boolean synonymBoostByPayload;
```

Comment:
I thought we switched the approach from a payload to boost attribute?  Besides; it's not clear we need this toggle at all since the user could arrange for this behavior simply by having the new DelimitedBoost filter thing in the chain.

Reply:
agreed and fixed!